server.go

```diff
@@ -5614,3 +5540,132 @@ func (s *server) AttemptRBFCloseUpdate(ctx context.Context,
 
 	return updates, nil
 }
+
+// getSelfNode returns the self node. It handles setting the node announcement,
+// signing it, and updating the source node in the graph.
```

Comment:
the comment should speak to the fact that config values that differ from defaults are prio'd, otherwise pre-existing settings found on disk are prio'd